customComponents/src/components/FooterLogo/index.js

```diff
@@ -1,12 +1,36 @@
 import React, { Component } from "react";
 import "./styles.scss";
+import Logo from "../Logo";
 
 export default class FooterLogo extends Component {
+  componentWillMount() {
+    this.handleCustomLogo();
+  }
+
+  handleCustomLogo = () => {
+    this.props.handleCustomLogo(true);
```

Comment:
I don't think this is the right abstraction. The child shouldn't need to tell the parent that it has a custom logo. Let's figure out a more straight forward way to address this.